Exercise_A/index.html

```diff
@@ -15,7 +15,7 @@
     />
   </head>
   <body>
```

Comment:
Hey Lukas, you did technically make two pages like asked but you have a button on each page that links you between the about.html and the index.html.

We wanted you to make an index.html with an image, when you click that image you should be redirected to a new site that contains a button that allows you to send and email.

Remember you can nest images within html tags.